### DISCUSSION TITLE
Followers bug

### DISCUSSION THREAD

**Discussion by shuvashish76** [community/community#8725] [Projects and Issues]
![Screenshot_2021-12-12-06-41-21](https://user-images.githubusercontent.com/48226312/145696770-917db18c-9575-4cac-bb64-fc1eb8f45ae2.jpeg)

Currently there are [3 followers](https://github.com/shuvashish76) but when you check for [followed accounts](https://github.com/shuvashish76?tab=followers) it shows 2 followers only.

Probably some deleted / deactivated account. Am I missing something or it's a GH bug?

**Comment by karthik558** [comment]
Ye , someone deleted there account or there account is banned by Github that's why its blank
